calendar.history

```diff
@@ -587,34 +593,34 @@
 10/18	Soviets announce their probe took photos of the Moon's far side, 1959
 10/18	The Grand Ole Opry opens its doors, 1925
 10/19	Boston Shoemakers form first American labor organization, 1648
-10/19	Mao Tse-tung establishes the People's Republic of China, 1949
+10/19	Mao Zedong establishes the People's Republic of China, 1949
 10/19	Napoleon's beaten army begins the long retreat from Moscow, 1812
-10/20	"Saturday Night Massacre", 1973
+10/20	"Saturday Night Massacre" resignation of US officials in protest
+	against attempt to dismiss Watergate prosecutor, 1973
 10/20	OPEC embargo, 1973
 10/21	Edison makes the first practical incandescent lamp, 1879
 10/21	Guggenheim Museum opens, 1959
 10/22	Laos gains independence from France, 1953
 10/23	Battle of Leyte Gulf begins, 1944
 10/23	Day of the 1956 revolution in Hungary
-10/23	Earth created at 6:30 AM, 4004BC.
 10/23	South African troops invade Angola in support of UNITA and FNLA, 1975
 10/23	Swallows leave Capistrano
 10/24	Founding of the United Nations, 1945
 10/25	The UN removes Taiwan and admits the People's Republic of China, 1971
 10/26	UN's World Health Organization declares smallpox eradicated, 1978
 10/27	New York's Boss Tweed is arrested on fraud charges, 1871
-10/27	The first New York Subway is opened, 1904
+10/27	The first New York subway line is opened, 1904
 10/27	Stock markets around the world crash because of fears of
 	a global economic meltdown, 1997
-10/28	Columbus discovers Cuba, 1492
+10/28	Columbus gets to Cuba about 4600 years too late to discover it, 1492
```

Comment:
https://en.wikipedia.org/wiki/History_of_Cuba#Pre-Columbian_(to_1500)

How about "Colombus reaches Cuba about 4600 years after its first settlers did, 1492" (trying to avoid "discover")

Reply:
I am fine with avoiding the word "discover" but I would like to avoid moving the focus from Columbus to the native populations: the calendar entry is about what Columbus did, not about who did the same thing before him.

I suggest "Columbus lands in Cuba" or, if you really want to state that someone was already there, "Columbus lands in Cuba: it is the first meeting between Europeans and Cuban natives".

Reply:
> I am fine with avoiding the word "discover" but I would like to avoid moving the focus from Columbus to the native populations: the calendar entry is about what Columbus did, not about who did the same thing before him.
> 
> I suggest "Columbus lands in Cuba" or, if you really want to state that someone was already there, "Columbus lands in Cuba: it is the first meeting between Europeans and Cuban natives".

The latter works for me.

Reply:
I think "Columbus lands in Cuba" is a fine balance. It's brief, it avoids saying he discovered it, if avoids implying that this is when the rampant colonization of the new world began, suppressing the native populations to the imperialist whims of Spain, and later others. Also, any other amplification of the first European may or may not be entirely accurate. We know the vikings were in the new world 400 years before. We know they sailed across at least part of the Atlantic by at least a hundred years before Columbus. etc. Keeping it simple, and generic gives the right information and right weight to it w/o getting bogged down in all these details. Plus, the data of first settlement is constantly changing, and 4600 years may well wind up being a late date in the future as the archeological record expands.

Reply:
Going for "Columbus gets to Cuba and meets inhabitants". Compare with "10/12	Bahama Natives discover Columbus of Europe lost on their shores" and "05/04	Christopher Columbus sets foot on Jamaica" for a range of wordings of similar events.